std/socket.d

```diff
@@ -86,7 +86,7 @@ else version (Posix)
 
     import core.stdc.errno;
 
-    enum socket_t : int32_t { init = -1 }
```

Comment:
It would seem this is a change of behaviour and is intended to be so:
```
import std.socket;
import std.stdio;
void main()
{
    socket_t sock;
    writefln("%d", sock); // prints -1
}
```

Reply:
`socket_t.init` will be equal to the first member value `socket_t._init`(-1), just as before.

Reply:
Ah, ok.